test/kubernetes_unittest.cc

```diff
@@ -33,6 +33,12 @@ class KubernetesTest : public ::testing::Test {
                                        is_deleted);
   }
 
+  std::vector<MetadataUpdater::ResourceMetadata> GetPodAndContainerMetadata(
```

Comment:
Any reason these accessors are not `static`?

Reply:
Fixed